datafusion/core/src/prelude.rs

```diff
@@ -30,12 +30,6 @@ pub use crate::execution::context::{SessionConfig, SessionContext};
 pub use crate::execution::options::{
     AvroReadOptions, CsvReadOptions, NdJsonReadOptions, ParquetReadOptions,
 };
-pub use crate::logical_plan::{
-    approx_percentile_cont, array, ascii, avg, bit_length, btrim, cast, character_length,
-    chr, coalesce, col, concat, concat_ws, count, create_udf, date_part, date_trunc,
-    digest, exists, from_unixtime, in_list, in_subquery, initcap, left, length, lit,
-    lower, lpad, ltrim, max, md5, min, not_exists, not_in_subquery, now, octet_length,
-    random, regexp_match, regexp_replace, repeat, replace, reverse, right, rpad, rtrim,
-    scalar_subquery, sha224, sha256, sha384, sha512, split_part, starts_with, strpos,
-    substr, sum, to_hex, translate, trim, upper, Column, Expr, JoinType, Partitioning,
-};
+
+pub use crate::logical_plan::{Column, Expr, JoinType, Partitioning};
```

Comment:
It would be nice to import `Expr`, `JoinType`, and `Partitioning` from `datafusion_expr` as well, and `Column` from `datafusion_common` to remove these remaining uses of the legacy `logical_plan` module.

Reply:
Ok, np